spec/source/chain/call_spec.rb

```diff
@@ -305,6 +305,6 @@ def baz
     chain = Solargraph::Source::SourceChainer.chain(source, Solargraph::Position.new(4, 11))
     type = chain.infer(api_map, Solargraph::Pin::ROOT_PIN, api_map.source_map('test.rb').locals)
     # @todo It would be more accurate to return `Enumerator<Array<Integer>>` here
-    expect(type.tag).to eq('Enumerator<String, Array<Integer>>')
+    expect(type.tag).to eq('Enumerator<undefined, String, Array<Integer>>')
```

Comment:
I'm not sure why this behavior change happened.  If you think it might be a concern, let me know - I can dig in on it with a fresh head later on.

Reply:
Ah, I'm getting a better picture.  A small fix gets us a real type here.